lab/AboutDialog/AboutDialog/AboutDialogViewModel.cs

```diff
@@ -0,0 +1,251 @@
+﻿using GraphQL.Client;
+using GraphQL.Common.Request;
+using GraphQL.Common.Response;
+using Microsoft.Toolkit.Parsers.Markdown;
+using Microsoft.Toolkit.Parsers.Markdown.Blocks;
+using Newtonsoft.Json.Linq;
+using RestSharp;
+using System;
+using System.Collections.ObjectModel;
+using System.Configuration;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using System.Windows.Controls;
+using System.Windows.Documents;
+
+namespace Sharpen
+{
+    internal class ConfigurationBaseViewModel : BindableBase
+    {
+        protected AboutDialogConfigurationSection configuration = ConfigurationManager.GetSection("aboutDialogConfigurationSection") as AboutDialogConfigurationSection;
```

Comment:
General remark that I forgot to give, sorry. Turn on C# 8.0 nullable reference types feature by adding `<nullable>true</nullable>` to the *csproj* file. Sharpen is compiled with this option so if you do it immediately in the lab it will be easier to integrate the code to the main source. Also, the compiler will give a warning here and it will be clear that the usage of the `as` operator here is probably not what was intended ;-)